proto/ethermint/evm/v1/evm.proto

```diff
@@ -24,6 +24,12 @@ message Params {
     (gogoproto.moretags) = "yaml:\"chain_config\"",
     (gogoproto.nullable) = false
   ];
+  // min gas denominator bounds the minimum gasUsed to be charge
```

Comment:
```suggestion
  // min gas denominator bounds the minimum gasUsed to be charged
```

Reply:
Good catch! will add it manually as I need to run the proto-gen with it to update doc 🙏